pyproject.toml

```diff
@@ -24,8 +24,7 @@ dependencies = [
     "pandas>=1.1.3;python_version<'3.10'",
     "pandas>=1.3.4;python_version>='3.10' and python_version<'3.11'",
     "pandas>=1.5.0;python_version>='3.11'",
-    "scikit-learn>=0.24,<2;python_version<='3.10'",
-    "scikit-learn>=1.1.3,<2;python_version>='3.11'",
+    "scikit-learn>0.24,<2",
```

Comment:
I think we have to cap for the minimum, otherwise when the script runs on python 3.11 it will try to install 0.24. 

I realize something was wrong with the script before and it was skipping the reinstallation of scikit-learn. I don't think 0.24 is actually available on 3.10 so I think we just have to update the requirements

Reply:
I will try to bump up the scikit version. Running into a similar issue to [this](https://github.com/scikit-learn/scikit-learn/issues/26858).